src/shared/components/mutationTable/column/ExonColumnFormatter.tsx

```diff
@@ -10,11 +10,12 @@ import styles from "./exon.module.scss";
 export default class ExonColumnFormatter {
     
     public static renderFunction(data:Mutation[],
-                                 genomeNexusCache:GenomeNexusCache|undefined) {
+                                 genomeNexusCache:GenomeNexusCache|undefined,
+                                 showTotalNrOfExons?:boolean) {
```

Comment:
don't be afraid of spelling out the variables, you can say `showTotalNumberOfExons` :)